package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@knocklabs/node",
-  "version": "0.4.17",
+  "version": "0.5.0",
```

Comment:
What's the thinking between making this a `minor` rather than a patch?

Reply:
We're introducing a new feature? We can make it a patch if you prefer. 👍